src/V128.cs

```diff
@@ -0,0 +1,99 @@
+﻿using System;
+using System.Runtime.CompilerServices;
+using System.Windows.Markup;
+
+namespace Wasmtime
+{
+    /// <summary>
+    /// A 128 bit value
+    /// </summary>
+    public struct V128
+    {
+        /// <summary>
+        /// Get a V128 with all bits set to 1
+        /// </summary>
+        public static readonly V128 AllBitsSet = new V128(
```

Comment:
Should we call this `MaxValue` instead?

Reply:
And perhaps also update the doc comment to indicate it is the maximum value of the `v128`?

Reply:
Although, the more I think about this, a `v128` is intended to be a vector of bits more akin to a SIMD register, so calling this the "maximum" value doesn't seem right.

I'm fine to leave this as-is.

Reply:
I was thinking of it more in terms of a vector of 16 bytes (or 4 ints/floats etc), rather than a single 128 bit value, so I personally think the current name makes more sense. Final decision is yours of course.